src/components/layout/TransactionComplete/TransactionComplete.tsx

```diff
@@ -18,37 +18,28 @@ export enum transaction_types {
   BUY_ARNS_NAME = 'Buy ArNS Name',
 }
 
-function TransactionSuccess({
+function TransactionComplete({
   transactionId = new ArweaveTransactionID(STUB_ARWEAVE_TXID),
-  state,
+  state, // eslint-disable-line
```

Comment:
why?

Reply:
oh. Wierd, thought I had to remove state, was having issue with ant card, removing something else must have fixed it.